Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -302,29 +302,18 @@ - (void)verificationWasCompleted
 
 - (void)presentAlertWithVerificationError:(NSError *)error
 {
-    UIAlertController *alertController;
```

Comment:
I now convert these errors in `TSAccountManager`, so this logic can be simplified.